### PR TITLE
Fix l10n on master

### PR DESCRIPTION
**Pull request by MorrisJobke**
- $app isn't useful for this case
- kill it with :fire:

cc @LukasReschke @DeepDiver1975

### CODE REVIEW

File: lib/private/l10n.php [hunk 1]
```diff
@@ -469,7 +460,7 @@ public static function findLanguage($app = null) {
 			return $default_language;
 		}
 
-		$lang = self::setLanguageFromRequest($app);
```
Comment (MorrisJobke) [inline lib/private/l10n.php]: This is the only usage for this parameter

**Comment by LukasReschke** [comment]
:+1:

**Comment by MorrisJobke** [comment]
cc @nickvergessen

**Comment by PVince81** [comment]
Makes sense :+1: 

I guess that was an outdated legacy use case ?

**Comment by MorrisJobke** [comment]
@PVince81 Mind to review this? (enabling enterprise_key causes the broken behaviour here)

**Comment by butonic** [comment]
:+1:

**Pull request merged**